Jenkinsfile_nightly

```diff
@@ -83,8 +84,11 @@ env.TEST_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
 // Other env variables needed for BEFTA.
 env.BEFTA_S2S_CLIENT_ID = "ccd_gw"
 env.ROLE_ASSIGNMENT_API_GATEWAY_S2S_CLIENT_ID = "ccd_data"
+env.BEFTA_S2S_CLIENT_ID_OF_CCD_DATA = "ccd_data"
+env.BEFTA_S2S_CLIENT_ID_OF_XUI_WEBAPP = "xui_webapp"
```

Comment:
Is this needed?

Reply:
resolved in fb506344bf94aaebeabf6903aab21df062468b0d